instat/static/InstatObject/R/Backend_Components/summary_functions.R

```diff
@@ -427,12 +427,15 @@ summary_mode <- function(x,...) {
   else return(out)
 }
 
-na_check <- function(x, na_type = "n", na_max_n = NULL, na_max_prop = NULL, na_FUN = NULL, ...) {
+na_check <- function(x, na_type = "n", na_max_n = NULL, na_max_prop = NULL, min_non_miss = NULL, na_FUN = NULL, ...) {

```

Comment:
I suggest changing to keep the same format as the other parameters.
```suggestion
na_check <- function(x, na_type = "n", na_max_n = NULL, na_max_prop = NULL, na_min_n = NULL, na_FUN = NULL, ...) {
```